hvplot/interactive.py

```diff
@@ -0,0 +1,362 @@
+            if self._depth == 0:
+                return self._clone(*args, **kwargs)
+            raise AttributeError
+        method = type(self._transform)(self._transform, self._method, accessor=True)
+        return self._clone(method(*args, **kwargs))
+
+    #----------------------------------------------------------------
+    # Interactive pipeline APIs
+    #----------------------------------------------------------------
+
+    # Builtin functions
+    def __abs__(self):
+        transform = type(self._transform)(self._transform, abs)
+        return self._clone(transform)
+    def __round__(self, ndigits=None):
+        args = () if ndigits is None else (ndigits,)
+        transform = type(self._transform)(self._transform, round_, *args)
+        return self._clone(transform)
+
+    # Unary operators
+    def __neg__(self):
+        transform = type(self._transform)(self._transform, operator.neg)
+        return self._clone(transform)
+    def __not__(self):
+        transform = type(self._transform)(self._transform, operator.not_)
+        return self._clone(transform)
+    def __invert__(self):
+        transform = type(self._transform)(self._transform, operator.inv)
+        return self._clone(transform)
+    def __pos__(self):
+        transform = type(self._transform)(self._transform, operator.pos)
+        return self._clone(transform)
+
+    # Binary operators
+    def __add__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.add, other)
+        return self._clone(transform)
+    def __and__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.and_, other)
+        return self._clone(transform)
+    def __div__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.div, other)
+        return self._clone(transform)
+    def __eq__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.eq, other)
+        return self._clone(transform)
+    def __floordiv__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.floordiv, other)
+        return self._clone(transform)
+    def __ge__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.ge, other)
+        return self._clone(transform)
+    def __gt__(self, other):
+        other = other._transform if isinstance(other, Interactive) else other
+        transform = type(self._transform)(self._transform, operator.gt, other)
```

Comment:
Is there a way to avoid specifying each of these delegations explicitly like this?  The answer may well be no, but it seems like a shame to have so much boilerplate code here, which makes it hard to see the real code, and would cover up any special processing that might be happening in one of these boilerplate methods.